p2p/protocol.go

```diff
@@ -72,6 +75,14 @@ type HeadBlock struct {
 	Time            uint64
 }
 
+type BlockHashEntry struct {
+	hash common.Hash
+	time time.Time
+}
+
+// DefaultTTL defines the time-to-live for block hash entries in blockHashes list.
+var DefaultTTL = 10 * time.Minute
```

Comment:
```suggestion
// blockHashTTL defines the time-to-live for block hash entries in blockHashes list.
var blockHashTTL = 10 * time.Minute
```

don't think we need to export this